clients/lsp-clangd.el

```diff
@@ -235,5 +235,24 @@ returned to avoid that the echo area grows uncomfortably."
         (car (s-lines (lsp--render-element contents)))))))
 
 
+(defun lsp-clangd-to-other ()
+  "Open the corresponding header/source file."
+  (interactive)
+  (if-let ((other-fname
+            ;; find-workspace might be too brittle if users make their own lsp IDs
+            ;; eg. clangd-remote
+            (with-lsp-workspace (lsp-find-workspace 'clangd)
```

Comment:
I don't think you need to do this, `lsp-send-request` can find the workspace that support the command. Or if the capability is not registered, it will send to all workspace, which is actually fine.